$NetBSD: patch-Makefile,v 1.6 2022/01/31 22:44:55 fcambus Exp $

- Honour CFLAGS and CXXFLAGS
- Do not hard-code -pthread, as it is handled by the pkgsrc infrastructure.

--- Makefile.orig	2022-01-23 08:19:40.000000000 +0000
+++ Makefile
@@ -34,14 +34,14 @@ endif
 
 # If you want to compile mold for debugging, invoke make as
 # `make CXXFLAGS=-g`.
-CFLAGS = -O2
-CXXFLAGS = -O2
+CFLAGS ?= -O2
+CXXFLAGS ?= -O2
 
 MOLD_CXXFLAGS = -std=c++20 -fno-exceptions -fno-unwind-tables \
                 -fno-asynchronous-unwind-tables -DMOLD_VERSION=\"1.0.2\" \
                 -DLIBDIR="\"$(LIBDIR)\""
 
-MOLD_LDFLAGS = -pthread -lz -lm
+MOLD_LDFLAGS = -lz -lm
 
 GIT_HASH = $(shell [ -d .git ] && git rev-parse HEAD)
 ifneq ($(GIT_HASH),)
